Pyrado/pyrado/algorithms/meta/idr.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c) 2020, Fabio Muratore, Honda Research Institute Europe GmbH, and
```

Comment:
Please leave IDR out for now, as I still consider it an ablation of SPDR. We can discuss about this.

Reply:
I added a comment on this fact but I'd leave it as a baseline for future research.